lldb/include/lldb/Target/Process.h

```diff
@@ -3139,6 +3146,7 @@ void PruneThreadPlans();
                            // m_currently_handling_do_on_removals are true,
                            // Resume will only request a resume, using this
                            // flag to check.
+  lldb::RunDirection m_last_run_direction;
```

Comment:
Feel free to initialize this here so we don't have to change the constructor:
```
lldb::RunDirection m_last_run_direction = lldb::eRunForward;
```

Reply:
Jim indicated that he doesn't want it initialized inline here.